docs/docs/tutorial/index.md

```diff
@@ -639,6 +639,19 @@ Submit the transaction through the already open WebSocket connection. Generate t
 cat tx-signed.json | jq -c '{tag: "NewTx", transaction: .}'
 ```
 
+Alternatively, you can submit the transaction using the HTTP API endpoint:
+
+```shell
+curl -X POST 127.0.0.1:4001/transaction \
+  --data @tx-signed.json \
+  --header "Content-Type: application/json"
```

Comment:
do we really need this?

other endpoint references do not mention this header, perhaps we can simplify the call as it should be handled by default.